src/Cthulhu.jl

```diff
@@ -197,21 +202,16 @@ function lookup(interp::CthulhuInterpreter, mi::MethodInstance, optimize::Bool)
     (codeinf, rt, infos, slottypes::Vector{Any})
 end
 
-module DInfo
-    @enum DebugInfo none compact source
-end
-using .DInfo: DebugInfo
-
 ##
 # _descend is the main driver function.
 # src/reflection.jl has the tools to discover methods
 # src/ui.jl provides the user facing interface to which _descend responds
 ##
-function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::MethodInstance; override::Union{Nothing, InferenceResult} = nothing, iswarn::Bool, params=current_params(), optimize::Bool=true, interruptexc::Bool=true, verbose=true, inline_cost::Bool=false, kwargs...)
-    debuginfo = DInfo.compact # default is compact debuginfo
-    if :debuginfo in keys(kwargs)
-        selected = kwargs[:debuginfo]
-        # TODO: respect default
+function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::MethodInstance;
+    override::Union{Nothing,InferenceResult}=nothing, debuginfo::Union{Symbol,DebugInfo}=DInfo.compact, # default is compact debuginfo
+    params=current_params(), optimize::Bool=true, interruptexc::Bool=true,
+    iswarn::Bool=false, verbose=true, inline_cost::Bool=false)
+    if isa(debuginfo, Symbol)
         debuginfo = getfield(DInfo, selected)::DebugInfo
```

Comment:
Where does `selected` get defined now?

Reply:
Good catch, it's just because I didn't run JET on this PR :P